packages/lib-classifier/src/store/WorkflowStore.spec.js

```diff
@@ -7,8 +7,6 @@ import {
 } from '../../test/factories'
 import stubPanoptesJs from '../../test/stubPanoptesJs'
 
-let rootStore
-
 const workflow = WorkflowFactory.build({
```

Comment:
Not a blocking comment for this PR, but we should be careful in future to scope variables like this one to the tests that use them. This should be declared inside the callback function that uses it.